Resources/views/Block/block_search_result.html.twig

```diff
@@ -12,43 +12,44 @@ file that was distributed with this source code.
 {% extends sonata_block.templates.block_base %}
 
 {% block block %}
-    <div class="col-md-3">
-        <div>
-            <h4>
-                {% if admin.hasRoute('list') %}
-                    <a href="{{ admin.generateUrl('list') }}">{{ admin.label|trans({}, admin.translationdomain) }}</a>
-                {% else %}
+    <div class="col-lg-4 col-md-6">
+        <div class="box">
+            <div class="box-header with-border">
+                <h3 class="box-title">
+                    {% set icon = settings.icon|default('') %}
```

Comment:
Looks at code of first box here: https://almsaeedstudio.com/themes/AdminLTE/pages/UI/general.html